tensorflow_quantum/core/src/circuit_parser_qsim.h

```diff
@@ -0,0 +1,42 @@
+/* Copyright 2020 The TensorFlow Quantum Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#ifndef TFQ_CORE_SRC_CIRCUIT_PARSER_QSIM_H_
+#define TFQ_CORE_SRC_CIRCUIT_PARSER_QSIM_H_
+
+#include <string>
+#include <vector>
+
+#include "../qsim/lib/circuit.h"
+#include "../qsim/lib/fuser.h"
+#include "../qsim/lib/gates_cirq.h"
```

Comment:
why `../qsim/*`, not `qsim/*`?

Reply:
qsim weirdly wouldn't let me do those imports. Wasn't sure how to fix right away. Do you have any ideas ?

Reply:
Let's defer to #272 . Thanks!